MediaBrowser.Model/Configuration/ServerConfiguration.cs

```diff
@@ -298,6 +298,18 @@ public ServerConfiguration()
         /// <value>The min resume duration seconds.</value>
         public int MinResumeDurationSeconds { get; set; } = 300;
 
+        /// <summary>
+        /// Gets or sets the minimum minutes of a book that must be played in order for playstate to be updated.
```

Comment:
What's wrong with the default that all other media uses?

Reply:
A 13 hour audiobook will be marked finished when theres about an hour left to listen to. It gets worse as the length gets longer. Percentages dont really work well.